### PR TITLE
Converts the organization controller to Kotlin.

### PR DESCRIPTION
**Pull request by geekbeast**
Behavior of API renames unchanged to maintain backwards compatibility with one exception-- getOrganizationEntitySets now returns an error instead of null when organization is not found.

Ignore JdbcConnection and DatasourcesApi for now.

### CODE REVIEW

File: api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+interface DatasourcesApi {
+    companion object {
+        // @formatter:off
+        const val SERVICE = "/datastore"
+        const val CONTROLLER = "/organization"
```
Comment (UnsungHero97) [inline api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt:15]: We have `/datastore/organizations` and with this `/datastore/organization`. Any reason why these have to be named so similarly? Why isn't the controller `/datasources`?

File: api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        const val BASE = SERVICE + CONTROLLER
+        // @formatter:on
+
+        const val ID = "id"
+        const val ID_PATH = "/{$ID}"
```
Comment (UnsungHero97) [inline api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt:20]: I think it'd be clearer if these are `ORG_ID`.

File: api/src/main/kotlin/com/openlattice/organizations/JdbcConnection.kt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.openlattice.organizations
+
+import com.fasterxml.jackson.annotation.JsonFilter
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
```
Comment (UnsungHero97) [inline api/src/main/kotlin/com/openlattice/organizations/JdbcConnection.kt:7]: Does this have to be `.*`?

File: api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt (hunk 1 above)
Comment (geekbeast) [reply api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt:15]: Ignore this.

File: api/src/main/kotlin/com/openlattice/organizations/JdbcConnection.kt (hunk 3 above)
Comment (geekbeast) [reply api/src/main/kotlin/com/openlattice/organizations/JdbcConnection.kt:7]: Ignore this.

File: api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt (hunk 2 above)
Comment (geekbeast) [reply api/src/main/kotlin/com/openlattice/organizations/DatasourcesApi.kt:20]: It's ID in organization api and these will be merged into that. Ignore this.

File: datastore/src/main/kotlin/com/openlattice/organizations/controllers/OrganizationsController.kt [hunk 4]
```diff
@@ -0,0 +1,902 @@
+package com.openlattice.datastore.services.com.openlattice.organizations.controllers
```
[the remark this answers is not in the document]
Comment (geekbeast) [reply datastore/src/main/kotlin/com/openlattice/organizations/controllers/OrganizationsController.kt:1]: Weird. Also weird that it compiled.

**Review by UnsungHero97 — APPROVED** [review]

**Review by makosblade — APPROVED** [review]

**Pull request merged**